apps/mobile/app/services/client/queries/timer/timer.ts

```diff
@@ -4,34 +4,36 @@ import { ITimerTimeslotParams } from "../../../interfaces/ITimer"
 
 type IGetTimerStatusParams = ITimerTimeslotParams & { authToken: string }
 
-const fetchTimerStatus = async (params: IGetTimerStatusParams) => {
+const fetchTimerStatus = async (params: IGetTimerStatusParams, isTimerRunning: boolean) => {
 	const { tenantId, organizationId, logType, authToken, employeeId } = params
 
-	const { data } = await getTimerStatusRequest(
-		{ source: "MOBILE", tenantId, organizationId },
-		authToken,
-	)
+	if (isTimerRunning) {
+		await syncTimeSlotRequest(
+			{
+				tenantId,
+				organizationId,
+				source: "MOBILE",
```

Comment:
@desperado1802 I think this is wrong. 

We have hard coded MOBILE source sync, we have to make it dynamic.

For Example:
If user started timer in WEB or DESKTOP or at other source we have to pass that value in source (If we don't find that value then only we have to pass MOBILE).


Take Reference from:
https://github.com/ever-co/ever-teams/blob/develop/apps/web/app/hooks/features/useTimer.ts#L238


So basically we have to at which source current timer is running, and we have to pass that value.